packages/spor-theme-react/src/components/select.ts

```diff
@@ -30,20 +30,19 @@ const baseStyleRoot: SystemStyleObject = {
 const baseStyleField: SystemStyleFunction = (props) => ({
   ...Input.baseStyle(props).field,
   appearance: "none",
-  pb: "1px",
   pt: "16px",
   "option, optgroup": {
     background: "alias.white",
   },
 });
 const baseStyleIcon: SystemStyleObject = {
-  width: "1.5rem",
-  height: "100%",
+  width: "30px",
+  height: "30px",
   insetEnd: "0.5rem",
   position: "relative",
   color: "currentColor",
   strokeLinecap: "round",
-  fontSize: "1.25rem",
+  fontSize: "18px",
```

Comment:
Vi vil helst bruke rem-units her, så ikonstørrelsen øker i takt med brukerens foretrukne tekststørrelse. Les litt mer om det konseptet her: https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

Reply:
Ahh skjønner